UnityProject/Assets/Scripts/Objects/Closets/AbandonedCrates.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections.Generic;
+using MiniGames;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace Objects.Closets
+{
+	public class AbandonedCrates : MonoBehaviour, IServerSpawn, ICheckedInteractable<HandApply>
+	{
+
+		[SerializeField] private MiniGameResultTracker miniGameTracker;
+		[SerializeField] private List<MiniGameModule> miniGameModules;
```

Comment:
I see that the crate is still linked directly to minigames. It's fine for now, just keep in mind that everything that has a minigame will need to do this too.